### PR TITLE
Adds drag'n drop and ReaderWriterZIP

### PR DESCRIPTION
**Pull request by jtorresfabra**
Also small refactor of ReaderWriterGLTF

### CODE REVIEW

**Comment by DavidPeicho** [comment]
Nice refactoring!
I should update the others examples with the new loading code whenever it is merged.

File: sources/osgPlugins/ReaderWriterGLTF.js [hunk 1]
```diff
@@ -124,218 +121,29 @@ GLTFLoader.prototype = {
...
-
-            } );
-            this._cachepromise[ uri ] = promise;
-
-            return promise;
-
-        }
-
-        var promiseFile = this.requestFileFromReader( urlOrFile, fileType );
+        var promiseFile = this.requestFileFromReader( uri, fileType );
```
Comment (cedricpinson) [inline sources/osgPlugins/ReaderWriterGLTF.js]: this method does not exist

**Comment by cedricpinson** [comment]
I thought about something like so I think it's good. We still need to discuss about some details.
About the zip plugin, I dont want the deps to be in the osgjs build, also be sure that if this code path is used, the user will be warned he has not the deps loaded in his page

File: sources/osgPlugins/ReaderWriterGLTF.js (hunk 1 above)
Comment (jtorresfabra) [inline sources/osgPlugins/ReaderWriterGLTF.js]: Oups, bug. But nope it should use the requestFile method ( https://github.com/cedricpinson/osgjs/pull/722/files#diff-29f7b843057ce77ec71c63808ab0c91bR57 ). 

The fileHelper is to read a list of files, like you have when drag'n dropping.

**Comment by jtorresfabra** [comment]
@cedricpinson Just to be sure, do you want JSZip in the vendors folder or just importing it as script tag pointing to a url in the index.html like https://github.com/cedricpinson/osgjs/blob/master/examples/pbr/index.html#L36

File: examples/dragndrop/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Gizmo</title>
```
Comment (cedricpinson) [inline examples/dragndrop/index.html]: example name :)

**Pull request merged**